docs/index.md

```diff
@@ -67,12 +66,12 @@ The easiest way to get started with Swarm is to use the
 [official Docker image](https://registry.hub.docker.com/_/swarm/).
 
 ```bash
-docker pull swarm
+$ docker pull swarm
```

Comment:
My bad on this one it is not on purpose (so you are right changing it @jamtur01) but for commands with placeholders (<...>), it is indeed on purpose :)